src/jobs/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ReportItem {
+  count: number;
+  ids: string[];
+  errors: string[];
+  errorCount: number;
+}
+
+export interface Report {
+  pausedApps: ReportItem | string;
+  expiredApps: ReportItem | string;
+  attestationNotifications: ReportItem | string;
+  expiryNotifications1: ReportItem | string;
+  expiryNotifications2: ReportItem | string;
+}
```

Comment:
It is very strange to me that you allow a job to return a simple string in the error case instead of a Report of what occured. Maybe with a JobSummary type that includes name, start time, etc. (see above comment) you will always have a nice object to return that includes (optional) error details and success status.

Reply:
iirc i did this because i was running into issues getting all the info i needed for a standard error object (see comment above) so yeah it's a lazy approach...i'm going to try another stab at this. Agree it's not a good implementation.

Reply:
Oh ignore this i read your comment wrong, no idea what my intention was here...will fix